things/api.py

```diff
@@ -476,30 +485,45 @@ def today(**kwargs):
     """
     Read Today's tasks into dicts.
 
-    Note: This might not produce desired results if the Things app hasn't
-    been opened yet today and the yellow "OK" button clicked for new tasks.
+    Note: Some tasks might be highlighted in yellow within the Things app.
```

Comment:
This entire note is difficult to read for me.

I propose to rewrite with three main messages:
1. The DB reflects the state the app was in when you last opened it.
2. Here, we additionally try to deduce what the yellow tasks right now would be if you opened the app.
3. Some yellow tasks are not handled yet: repeats, …

Reply:
I gave it a try.